Index: memcached/testapp.c
===================================================================
--- memcached.orig/testapp.c
+++ memcached/testapp.c
@@ -2331,8 +2331,6 @@ int main(int argc, char **argv)
     int ii = 0, num_cases = 0;
 #ifdef TLS
     if (getenv("SSL_TEST") != NULL) {
-        SSLeay_add_ssl_algorithms();
-        SSL_load_error_strings();
         enable_ssl = true;
     }
 #endif
Index: memcached/tls.c
===================================================================
--- memcached.orig/tls.c
+++ memcached/tls.c
@@ -128,7 +128,7 @@ const unsigned char *ssl_get_peer_cn(con
     }
 
     // can't use get0 to avoid getting a reference since that requires 3.0.0+
-    X509 *cert = SSL_get_peer_certificate(c->ssl);
+    X509 *cert = SSL_get1_peer_certificate(c->ssl);
     if (cert == NULL) {
         return NULL;
     }
